src/microdot/websocket.py

```diff
@@ -160,7 +160,12 @@ async def _read_frame(self):
             raise WebSocketError('Message too large')
         if has_mask:  # pragma: no cover
             mask = await self.request.sock[0].read(4)
-        payload = await self.request.sock[0].read(length)
+        payload = bytearray()
+        while len(payload) < length:
+            chunk = await self.request.sock[0].read(length - len(payload))
+            if not chunk:
```

Comment:
It does not appear that `socket.read()` ever returns `None`. At least the docs do not mention it. What is the purpose of the check?